api/journaling_api.md

```diff
@@ -303,13 +303,20 @@ Link: </events/organizations/23294/integrations/54108/f89067f2-0d50-4bb2-bf78-20
 
 ### Limiting the size of the batch
 
-Depending on the frequency of the events in your registration, the number of events returned in a single response batch varies. A batch of events will always have one event but there is no upper limit to the number of events that can be returned in a single batch. In case you wish to set an upper bound, you can supply the `limit` query parameter with the maximum number of events that may be returned by the API.
+When events are created at a high frequency, Journal persists groups of events in its storages; 
```

Comment:
storages (plural) is not a word. You can use storage (singular), or storage units (plural)

Reply:
done see https://github.com/AdobeDocs/adobeio-events/pull/77/commits/a4d1499d21be759ed159ab3cc1d367af9183b405